packages/hash/frontend/src/components/GlideGlid/utils/draw-vertical-line.ts

```diff
@@ -0,0 +1,32 @@
+import { CustomCell } from "@glideapps/glide-data-grid";
+import type { DrawArgs } from "@glideapps/glide-data-grid/dist/ts/data-grid/cells/cell-types";
+import { getYCenter } from "../utils";
+
+export type VerticalLineDir = "up" | "down" | "full" | "none";
```

Comment:
Hmm, It'll be used in other cells as well pretty soon. But I get your point. I'll call this file `draw-vertical-indentation-line` and type `VerticalIndentationLineDir`. Because it's not just a "line", it's specifically "indentation line"

Reply:
It's actually the strings I think could have better names – i.e – "first" | "last" | "middle". I don't think "none" is ever used?

Reply:
You're right, updated it!